### DISCUSSION TITLE
How to provide environment variables to configure markdown and markdown_inline plugin compilation?

### DISCUSSION THREAD

**Discussion by powerman** [nvim-treesitter/nvim-treesitter#7335] [Q&A]
There are a lot of env vars which may be used to setup this plugin while compilation: https://github.com/tree-sitter-grammars/tree-sitter-markdown?tab=readme-ov-file#extensions

I've tried to set them using `:lua vim.env.ALL_EXTENSIONS='true'` and then run `:TSInstall markdown` and `:TSInstall markdown_inline` and restart Neovim, but didn't notice any changes in syntax tree output (`:InspectTree`), so I suppose these env vars wasn't actually used.

https://github.com/tree-sitter-grammars/tree-sitter-markdown/blob/1c8dea73bc0c996d92dd9ebc30dd388716b1c5db/common/common.js#L1-L10

**Comment by clason** [comment]
You can't, right now. If you need non-default features, you need to build and install the parsers yourself.
